ui/app/components/send_/send.utils.js

```diff
@@ -147,6 +148,37 @@ function getAmountErrorObject ({
   return { amount: amountError }
 }
 
+function getGasFeeErrorObject ({
+  amount,
+  amountConversionRate,
+  balance,
+  conversionRate,
+  gasTotal,
+  primaryCurrency,
+  selectedToken,
+  tokenBalance,
+}) {
+  let insufficientFunds = false
+  if (gasTotal && conversionRate) {
+    insufficientFunds = !isBalanceSufficient({
+      amount: '0x0',
+      amountConversionRate,
+      balance,
+      conversionRate,
+      gasTotal,
+      primaryCurrency,
+    })
+  }
+
+  let gasFeeError = null
+
+  if (insufficientFunds) {
+    gasFeeError = INSUFFICIENT_FUNDS_ERROR
+  }
+
+  return { gasFee: gasFeeError }
```

Comment:
I think this would be more readable:
```
let gasFeeError = null

if (gasTotal && conversionRate) {
  const insufficientFunds = !isBalanceSufficient({
    amount: '0x0',
    amountConversionRate,
    balance,
    conversionRate,
    gasTotal,
    primaryCurrency,
  })

  if (insufficientFunds) {
    gasFeeError = INSUFFICIENT_FUNDS_ERROR
  }
}

return { gasFee: gasFeeError }
```

And I don't think `selectedToken` or `tokenBalance` are being used. You might need to configure your linter to show unused variables. VSCode makes this easy!

Reply:
Fixed these with latest push.